ocaml/driver/main_args.ml

```diff
@@ -2155,6 +2264,60 @@ module Default = struct
       set Flambda2.Debug.concrete_types_only_on_canonicals
     let _no_flambda2_debug_concrete_types_only_on_canonicals =
       clear Flambda2.Debug.concrete_types_only_on_canonicals
+
+    let _flambda2_inline_max_depth spec =
+      Int_arg_helper.parse spec
+        "Syntax: -flambda2-inline-max-depth <int> | <round>=<int>[,...]"
+        Flambda2.Inlining.max_depth
+
+    let _flambda2_inline_alloc_cost spec =
```

Comment:
(The various "cost" functions could be factored out as above,
but I am not positive it is worthwhile.)

Reply:
Yeah I'm not sure it's worth it here.